styles.css

```diff
@@ -81,225 +297,255 @@ body {
-    grid-template-columns: repeat(1fr);
-    margin-top: 50px;
-    margin-left: 4vw;
-    margin-right: 4vw;
-    gap: 50px;
-  }
-
-  .Tonic {
-    color: #172b4d;
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 700;
-    font-size: 32px;
-    line-height: 44px;
-  }
-
-  .canopy {
+  .navlinks {
+    list-style-type: none;
     display: flex;
     flex-direction: row;
     align-items: center;
-    gap: 12px;
+    padding: 0;
+    gap: 8px;
   }
 
-  .canopi {
+  .navlinks li a {
     font-family: 'Poppins', Arial, Helvetica, sans-serif;
     font-style: normal;
-    font-weight: 600;
-    font-size: 13px;
-    line-height: 16px;
-    color: #344563;
-  }
-
-  .backend {
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 600;
-    font-size: 13px;
-    line-height: 16px;
-    color: #7a869a;
-  }
-
-  .fiftin {
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 600;
-    font-size: 13px;
-    line-height: 16px;
-    color: #7a869a;
-  }
-
-  .art {
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 400;
+    font-weight: 500;
     font-size: 15px;
-    line-height: 24px;
+    line-height: 20px;
     color: #344563;
+    margin-right: 10vw;
+    text-decoration: none;
   }
 
-  .frontend {
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 500;
-    font-size: 12px;
-    line-height: 16px;
-    color: #6070ff;
-    list-style-type: none;
-    display: flex;
-    gap: 16px;
-    padding-left: 0;
-  }
-
-  .frontend li {
-    background: #ebebff;
-    padding: 4px 12px;
-    border-radius: 8px;
-  }
-
-  .seeproject {
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 500;
-    font-size: 17px;
-    line-height: 24px;
-    color: #396df2;
-    border: 1px solid #6070ff;
-    border-radius: 8px;
-    background-color: inherit;
-    padding: 12px;
-    gap: 10px;
-    margin-top: 16px;
-    margin-bottom: 40px;
+  nav {
+    display: none;
   }
 
   .card1 {
     display: flex;
-    flex-direction: column;
-    justify-content: center;
+    flex-direction: row;
+    justify-content: space-around;
     padding: 16px;
     border-radius: 16px;
     border: solid #e5e5e5 1px;
     background-color: #fff;
+    gap: 24px;
   }
 
   .about {
+    display: flex;
     background-color: #fff;
     border-radius: 0 100px 0 0 / 100px;
     padding-top: 8%;
     margin-top: 16%;
   }
 
   #about {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
     margin: 4vw;
   }
 
-  .aboutmyself {
-    margin-left: 8vw;
-    margin-right: 8vw;
+  #prlanguages {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-right: 14vw;
+  }
+
+  #prlanguages li {
+    display: flex;
+    flex-direction: column;
```

Comment:
- [ ] The languages squares have a similar dimention, please 🙏🏽 adjust the width to match the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/220824757-3c5957dd-9a03-4553-a4c7-d139052ed56a.png) | ![image](https://user-images.githubusercontent.com/58642949/220824796-98019530-628d-49e6-927b-15e0778de732.png) |